java-checks/src/main/java/org/sonar/java/checks/BadConstantName_S00115_Check.java

```diff
@@ -99,7 +100,8 @@ private void checkName(VariableTree variableTree) {
   private boolean isStaticFinal(VariableTree variableTree) {
     boolean isStatic = false;
     boolean isFinal = false;
-    for (Modifier modifier : variableTree.modifiers().modifiers()) {
+    for (ModifierKeywordTree modifierKeywordTree : variableTree.modifiers().modifiers()) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Impossible cast from org.sonar.plugins.java.api.tree.Modifier to org.sonar.plugins.java.api.tree.ModifierKeywordTree in org.sonar.java.checks.BadConstantName_S00115_Check.isStaticFinal(VariableTree) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=findbugs%3ABC_IMPOSSIBLE_CAST)